modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormTemplateContextFactoryImpl.java

```diff
@@ -227,20 +229,36 @@ protected String getDDMFormContextProviderServletURL() {
 	}
 
 	protected String getRequiredFieldsWarningMessageHTML(
-		ResourceBundle resourceBundle) {
+		ResourceBundle resourceBundle,
+		DDMFormRenderingContext ddmFormRenderingContext) {
 
 		StringBundler sb = new StringBundler(3);
 
 		sb.append("<label class=\"required-warning\">");
 		sb.append(
 			LanguageUtil.format(
 				resourceBundle, "all-fields-marked-with-x-are-required",
-				"<i class=\"icon-asterisk text-warning\"></i>", false));
+				getRequiredMarkTagHTML(ddmFormRenderingContext), false));
 		sb.append("</label>");
 
 		return sb.toString();
 	}
 
+	protected String getRequiredMarkTagHTML(
+		DDMFormRenderingContext ddmFormRenderingContext) {
+
```

Comment:
Hey @pedroqueiroz  Could you please pass the `HttpServletRequest` instead of the `DDMFormRenderingContext` as the method argument? Thx